Jenkinsfile

```diff
@@ -333,6 +333,8 @@ pipeline {
                             sh '''
                             echo "Ensuring container killed."
                             docker rm -vf docker-pr$BUILD_NUMBER || true
```

Comment:
Looks like the new lines are to replace this one, so I think this one can be removed

Reply:
Ideally we would keep both of these together and refactor them in a cleanup function.

Reply:
But this stage doesn't start a container named `docker-pr$BUILD_NUMBER` without a `-x` suffix

Reply:
@thaJeztah Yes, I know. I wanted to keep it so that whoever refactors this Jenkinsfile mess doesn't forget to keep the two together. But fine i'll remove it.

Reply:
Each stage should have a cleanup step, but this stage is different from the others, so I think it's better to not have code in there that's not used